src/Compilers/VisualBasic/Portable/Parser/ParseStatement.vb

```diff
@@ -850,12 +845,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                 optionalAsClause = SyntaxFactory.SimpleAsClause([As], Nothing, Type)
             End If ' Else if "As" is not present, the error falls out as a "Syntax error" IN the caller
 
-            Dim names = _pool.AllocateSeparated(Of ModifiedIdentifierSyntax)()
```

Comment:
>() [](http://example.com/codeflow?start=76&length=2)

Please keep the argument list.

Reply:
Correct in recent push.

Reply:
>> Please keep the argument list.

> Correct in recent push.

It looks like the argument list for invocation of ```AllocateSeparated``` is still missing.